azure-pipelines.yml

```diff
@@ -16,6 +16,11 @@ jobs:
   variables:
     display: ":99"
   steps:
+  - task: InstallSSHKey@0
+    inputs:
+      hostName: "github.com,192.30.253.112 ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAq2A7hRGmdnm9tUDbO9IDSwBK6TbQa+PXYPCPy6rbTrTtw7PHkccKrpp0yVhp5HdEIcKr6pLlVDBfOLX9QUsyCOV0wzfjIJNlGEYsdlLJizHhbn2mUjvSAHQqZETYP81eFzLQNnPHt4EVVUh7VfDESU84KezmD5QlWpXLmvU31/yMf+Se8xhHTvKSCZIFImWwoG6mbUoWf9nzpIoaSjB+weqqUUmpaaasXVal72J+UX2B+2RPW3RcT0eOzQgqlJL3RKrTJvdsjE3JEAvGq3lGHSZXy28G3skua2SmVi/w4yCE6gbODqnTWlg7+wC604ydGXA8VJiS5ap43JXiUFFAaQ=="
```

Comment:
out of curiousity, what was this about?

Reply:
Oh, this was what I had to do to install what-the-line from a private git dependency. This sets up `~/.ssh` with GitHub in its known_hosts file and a temporary SSH key I generated in the Azure Pipelines builds' "secret files stash" so that `apm ci` succeeds.